tensorboard/components/tf_paginated_view/tf-paginated-view2.html

```diff
@@ -256,16 +256,30 @@
 
     behaviors: [
       tf_dashboard_common.ArrayUpdateHelper,
-      Polymer.Templatizer,
     ],
 
-    ready() {
-      // Template instance props that should be excluded from forwarding
-     this._instanceProps = {
-       [this.as]: true,
-     };
-
-      this.templatize(this.$.template.querySelector('template'));
+    _ensureTemplatized() {
+      // Polymer is not ready (and props/DOM for the components are not
+      // populated)
+      if (!this.isAttached) return false;
+
+      if (!this._ctor) {
+        const templateNode = this.$.template.assignedElements()[0];
+        this._ctor = Polymer.Templatize.templatize(templateNode, this, {
+            parentModel: true,
+            instanceProps: {
+              [this.as]: true,
+              active: true,
+            },
+            forwardHostProp: function(prop, value) {
+              // Polymer uses `call` to invoke this method.
```

Comment:
Not sure what this is supposed to mean. It looks like the function body
expects the `this` context to be the element itself; if so, can’t we
just use an arrow function and drop the comment? If I’m
misunderstanding, can you please clarify?